src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -251,6 +253,10 @@ float MulticopterLandDetector::_get_max_altitude()
 	/* TODO: add a meaningful altitude */
 	float valid_altitude_max = _param_lndmc_alt_max.get();
 
+	if (valid_altitude_max < 0.0f) {
+		return INFINITY;
```

Comment:
This function returns `bool`, returning INFINITY seems wrong.

Reply:
Thanks for your review @dakejahl !  I might be mistaken, but I think it returns float, (line 251), so I think we're in good shape here.

Reply:
ahh my bad, apparently I suck at github diffs. Just realized it shows 
`@@ -251,6 +253,10 @@ float MulticopterLandDetector::_get_max_altitude()`